app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -54,6 +61,12 @@ export default class AMRHeatmapVis extends React.Component {
     ) {
       this.heatmap.updateColumnMetadata(this.getSelectedMetadataFields());
     }
+    if (nodeHoverTriggered !== prevState.nodeHoverTriggered) {
+      logAnalyticsEvent("AMRHeatmapVis_heatmap-node-hover_triggered");
```

Comment:
`AMRHeatmapVis_heatmap-node_hovered` and same below.

Also, maybe it should be `state.nodeHovered` for consistency.